src/edge/views.py

```diff
@@ -1,29 +1,51 @@
 import json
+import os
 import random
+import tempfile
 
 from django.core.urlresolvers import reverse
-from django.http import HttpResponse
+from django.http import HttpResponse, JsonResponse
 from django.views.generic.base import View
+from django.views.decorators.http import require_http_methods
 from django.shortcuts import get_object_or_404
 from django.db.models import Q
 from django.db import transaction
 
 from edge.models import Fragment, Genome, Operation
 from edge.io import IO
+from edge import import_gff
+from edge.tasks import build_genome_blastdb
 
 
 def genome_export(request, genome_id):
     get_genome_or_404(genome_id)
     io = IO(Genome.objects.get(pk=genome_id))
-
     response = HttpResponse(content_type='text/plain')
     response['Content-Disposition'] = "attachment; filename=\"g%s.gff\"" % genome_id
     io.to_gff_file(response)
     return response
 
 
+@require_http_methods(["POST"])
+def genome_import(request):
+    res = {
+      'imported_genomes': [],
+    }
+    for name in request.FILES:
+        with tempfile.NamedTemporaryFile(mode='rw+b', delete=False) as gff:
+            gff.write(request.FILES.get(name).read())
```

Comment:
using `.read()` can cause OOM errors for large files. Since GFFs can be large, I suggest doing the django-recommended best-practice and using `UploadedFile.chunks`. See [the docs](https://docs.djangoproject.com/en/1.11/topics/http/file-uploads/#basic-file-uploads) for an example.